assets/frontend/js/globalpayments-secure-payment-fields.js

```diff
@@ -580,6 +584,13 @@
 					target: helper.getSubmitButtonTargetSelector( this.id )
 				}
 			};
+			if ( 'globalpayments_gpapi' == this.id ) {
```

Comment:
Please replace the condition with:
if (this.fieldOptions.hasOwnProperty('card-holder-name-field'))

This way, the code will work for other gateways adding the field.